### DISCUSSION TITLE
No ability to choose json type for the schema for query results

### DISCUSSION THREAD

**Discussion by garyboyes** [Budibase/budibase#8697] [Help]
I'm attempting to read in data from a table via a query, where it has a column of json data. When getting the results of the query, the json column is output as object Object, and the schema has no option to choose json as a value type, only text, number, boolean or datetime - is there a reason for json as a type support not being available?

Have read some older discussions around using transformers as a workaround, but it would be great to just natively support json inline, instead of needing to use workarounds

**Comment by melohagan** [comment]
Hey @garyboyes 

There is an open issue here that you can follow: https://github.com/Budibase/budibase/issues/3382 (and https://github.com/Budibase/budibase/issues/8389)
